src/javascripts/components/auth/auth.js

```diff
@@ -0,0 +1,14 @@
+
+import firebase from 'firebase/app';
+import 'firebase/auth';
+
+const signMeIn = () => {
```

Comment:
since you moved the code for login into the navbar lets delete this file and put this code in myNavbarjs

Reply:
breaks code

Reply:
yep ok seeing this now too.  its because the navbar is created dynamically.  we will leave these as is